build/common.sh

```diff
@@ -1256,10 +1288,6 @@ function kube::release::gcs::verify_ci_ge() {
       greater=false
     elif [[ "${version_prerelease_rev}" -gt "${gcs_version_prerelease_rev}" ]]; then
       : # fall out
-    elif [[ "${version_patch}" -lt "${gcs_version_patch}" ]]; then
-      greater=false
-    elif [[ "${version_patch}" -gt "${gcs_version_patch}" ]]; then
-      : # fall out
```

Comment:
These lines got accidentally copy-pasted in; they're identical to comparisons above, so should be removed.